src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/StacksTests.cs

```diff
@@ -40,6 +40,17 @@ public class StacksTests
         private const string NativeFrame = "[NativeFrame]";
         private const string ExpectedThreadName = "TestThread";
 
+        private static MethodInfo GetMethodInfo(string methodName)
+        {
+            // Strip off any generic type information. Pseudo frames (eg. [NativeFrame]) will return null anyways.
+            if (methodName.Contains('['))
+            {
+                methodName = methodName[..methodName.IndexOf('[')];
+            }
+
```

Comment:
nit: Rather than letting psuedo frames enter the below check and rely on it returning null, let's add an explicit check to make the intention clear.

e.g.

```csharp
// Return null on psuedo frames (e.g. [NativeFrame])
if (methodName.Length == 0)
{
  return null;
}

```